scripts/xtensa-build-all.sh

```diff
@@ -19,7 +19,8 @@ then
 	echo "       [-u] Force UP ARCH"
 	echo "       [-d] Enable debug build"
 	echo "       [-c] Configure defconfig"
-	echo "       [-j [n]] Set number of make build jobs. Infinite jobs with no arg."
+	echo "       [-j [n]] Set number of make build jobs." \
```

Comment:
With your changes it works for example like that:
`scripts/xtensa-build-all.sh apl` -> `make bin -j$(nproc)`
`scripts/xtensa-build-all.sh -j apl` -> `make bin -j`
`scripts/xtensa-build-all.sh -jX apl` -> `make bin -jX`

So with noarg it's still `<infinite>` jobs. Previously `-j` arguments worked exactly like `make`, now it will use `nproc`, when there is no `-j` arg.

Reply:
Pushed an updated comment